Examples/Movies/MovieCell.js

```diff
@@ -22,7 +22,7 @@ var {
   StyleSheet,
   Text,
   TouchableHighlight,
-  View
+  View,
```

Comment:
@sebmck - it is, but as a matter of style an consistency across the project it is used. See: https://github.com/elierotenberg/coding-styles/blob/master/es6.md

Related: if anyone thinks it's worthwhile, I could draft a code style guide for react-native in order to make things more clear cc @vjeux